src/sync/index.js

```diff
@@ -481,13 +469,24 @@ async function getInsertBlockPromises(db, startBlock, endBlock) {
   return { insertBlockPromises, endBlockTime: blockTime };
 }
 
+function calculateSyncPercent(blockTime) {
+  let syncPercent = 100;
+  const block0Time = getBlockChainConstants.BLOCK_0_TIMESTAMP;
+  // if blockTime is 10 min behind, we are not fully synced
+  if (blockTime < moment().unix() - 600) {
+    syncPercent = Math.floor((blockTime - block0Time)/(moment().unix() - block0Time)*100);
```

Comment:
create reusable var for `moment().unix()`

Reply:
done